CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+## 1.7.0 (May 12, 2020)
```

Comment:
According to changes my suggestion is up version to 2.0.0. Or all this changes are backward compatible?